src/nvim/globals.h

```diff
@@ -789,10 +789,9 @@ EXTERN char *escape_chars INIT(= " \t\\\"|");  // need backslash in cmd line
 
 EXTERN bool keep_help_flag INIT(= false);  // doing :ta from help file
 
-// When a string option is NULL (which only happens in out-of-memory
-// situations), it is set to empty_option, to avoid having to check for NULL
-// everywhere.
-EXTERN char *empty_option INIT(= "");
+// When a string option is NULL (which only happens in out-of-memory situations), it is set to
+// empty_string_option, to avoid having to check for NULL everywhere.
+EXTERN char *empty_string_option INIT(= "");
```

Comment:
Potential future improvement: is this even possible in Nvim? Is the OOM being referred-to happening during startup, or could it happen after startup? Does it even matter?

Reply:
I'm not sure, I would have to look into it. Either way, we could add just ref-counted Strings which removes the need for this